storybook/src/utils/Margin/Margin.stories.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Paragraph } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+import type { HTMLAttributes } from 'react'
+
+const lengthOptions = ['xs', 'sm', 'md', 'lg', 'xl'] as const
+
+type MarginProps = {
+  length: (typeof lengthOptions)[number]
+} & HTMLAttributes<HTMLElement>
+
+const Margin = ({ children, length }: MarginProps) => <span className={`ams-mb--${length}`}>{children}</span>
+
+const render = ({ length }: MarginProps) => (
+  <div>
```

Comment:
We can render an array here, so we don't have to wrap it in a `div`. Same goes for the visually hidden docs.

Reply:
Had tried that, but not correctly maybe. Works now.